html/semantics/interfaces.html

```diff
@@ -27,7 +33,9 @@
 }
 
 elements.forEach(function(a) {
-  do_test(a[0], a[1]);
-  do_test(a[0].toUpperCase(), a[1]);
+  do_test(a[0], a[1], true);
+  if (a[0] === a[0].toLowerCase()) {
```

Comment:
I don't know anything about custom elements, so I guess I'll pass on this.  LGTM with the above comments addressed (with the caveat that I don't understand the bit about custom elements).